### DISCUSSION TITLE
Scott Chacon - GitHub, GitBulter

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#286] [Announcements]
- https://scottchacon.com/
- https://github.com/schacon
- https://twitter.com/chacon?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor
- https://www.linkedin.com/in/schacon/?originalSubdomain=de

### Git/GitHub

- [x] How did the idea for GitHub start? Did you ever think it would be where it is today?
- [x] In the early days when git wasn't as popular what work did you have to do to make it so ubiquitous?
- [x] I noticed you name is all throughout the git documentation and book. Why did you choose to write all of this? 
- [x] What's your favorite lesser known part of git?
- [x] Do you think knowing so much about git lets you do interesting things with it?

### Git Butler

- [x] What is git butler?
- [x] why build another ui when the git cli works?
- [x] What are the benefits of using virtual branches?
- [x] Are merge conflicts harder to deal with?
- [x] What is the timeline and how does it work?
- [x] Git butler seems unique in the way that it build a new workflow on top of git. Are there any other tools you know of that do that also?
- [x] What are some ways that AI is integrated into the tool? What other ways do you think it could be used?

### Future

- [x] What do you think the future of version control systems is?

**Comment by just-be-dev** [comment]
https://www.inkandswitch.com/patchwork/notebook/
